src/main/java/io/jenkins/plugins/sshbuildagents/ssh/mina/KeyAlgorithmManagerImpl.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright The Original Author or Authors
+ * SPDX-License-Identifier: MIT
+ */
+package io.jenkins.plugins.sshbuildagents.ssh.mina;
+
+import io.jenkins.plugins.sshbuildagents.ssh.KeyAlgorithm;
+import io.jenkins.plugins.sshbuildagents.ssh.KeyAlgorithmManager;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Implementation of KeyAlgorithmManager that provides a list of supported key algorithms. TODO
+ * Implement a proper key algorithm manager that returns supported algorithms.
```

Comment:
what will be the goal of this? 
Maybe it could be just as easy as delegates this Mina as anyway, we will depend on his validation mechanism.